e2e/docker_compose_redpanda.yaml

```diff
@@ -3,6 +3,7 @@ version: "3"
 services:
   # is redpanda: reusing names to not break kafka integration tests
   kafka:
+    image: docker.redpanda.com/redpandadata/redpanda:${REDPANDA_VERSION:-v24.1.8}
```

Comment:
that's a different version than the one used in the E2E workflow?

Reply:
fixed

Reply:
See earlier comment on latest versions